pages/news/2020-XX-XX-concrete-action.html

```diff
@@ -0,0 +1,65 @@
+We are working to migrate our code away from these words, even in many cases where the particular meaning of "master" was never explicitly associated with a problematic context.
+
+It’s important to note that there will be some cases where the word will remain.
+Many DJ controllers have buttons marked "master" on them, and we need to refer to those buttons by name.
+There are also the audio engineering concepts of a "master tape" or "audio mastering", so there may be areas of our documentation or code that would be less understandable if we tried to use synonyms.
+But in situations where "master" is used in code, we commit to changing the language we use by the end of the year.
+
+But changing language is not enough.
+
+## Building A More Diverse Community
+
+As an entirely online community that allows anyone to participate with a pseudonym, we don't know the exact demographic makeup of our contributors.
+Our impression, however, is that the Mixxx contributor base is largely white, male, and either from the US or Western Europe.
+Employers have a tacit (and sometimes explicit) expectation that prospective employees "prove themselves" by contributing to open source projects. Companies also often value referrals, which increase the likelihood that new hires will be similar to previous hires, thus perpetuating homogeneity.
+We want to break this cycle and ensure that the door to our open source network is open to everyone.
+It is our goal to make both the Mixxx community and the software we make as inclusive, accessible and welcoming as possible.
+As such, it is our responsibility to actively seek out and promote the participation of groups who are underrepresented in our community.
+
+Research shows that blind admission and hiring tends to result in less diverse outcomes.
+To that end, we plan to participate in the [Outreachy](https://www.outreachy.org/) program and have a paid internship position by December 2020, the next available cycle.
+Outreachy’s stated goal is to provide "internships to work in open source and free software, for women (both cis and trans), trans men, and genderqueer people" and "residents and nationals of the United States of any gender who are Black/African American, Hispanic/Latin@, Native American/American Indian, Alaska Native, Native Hawaiian, or Pacific Islander."
+They encourage "anyone who faces under-representation, systemic bias, or discrimination in the technology industry of their country" to apply.
+
+Our goal is that this Outreachy intern will be one of an increasing number of valued contributors from more diverse backgrounds that will help us be a healthier, more sustainable open source project.
+With your help, we intend to continue the internship process semiannually for at least 2 cycles, or how ever many we are able to raise funding for, and at that time assess if we should continue with the program or if our diversification efforts would be better served with a new approach.
```

Comment:
How to we fund the second cycle of we don't get enough donations? 

And it's also possible that we stop after 2 cycles if our assessment determined we should rather do something else right? But it also says we participate how every many cycles we are able to raise funding for. That's a contradiction.